### PR TITLE
added media que and grid layout

### PR DESCRIPTION
**Pull request by ihorandrianov**
- [DEMO LINK](https://ihorandrianov.github.io/layout_catalog_grid/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="/">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
```
Comment (grafit-off) [inline src/index.html:22]: The wrong path in link, you should use "#"

File: src/index.html [hunk 2]
```diff
@@ -17,35 +17,35 @@
...
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="/" class="nav__link">Apple</a>
```
Comment (grafit-off) [inline src/index.html:27]: Next time use "#" for this case

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,22 +1,30 @@
...
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
+  }
+
+  margin: 0 5%;
```
Comment (grafit-off) [inline src/styles/_catalog.scss:18]: You can't set margins to BEM block ❗

**Review by grafit-off — APPROVED** [review]
Keep going 🔥

File: src/index.html (hunk 1 above)
Comment (Denchepornyuk) [reply src/index.html:22]: agree

File: src/styles/_catalog.scss (hunk 3 above)
Comment (Denchepornyuk) [reply src/styles/_catalog.scss:18]: yep, you can use margin for BEM block

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]
fix comments from @grafit-off

**Review by Anna-Tkachenko — APPROVED** [review]
Good job!

**Pull request closed without merging**